.eleventy.js

```diff
@@ -15,9 +15,9 @@ module.exports = function(eleventyConfig) {
     return {
       dir: {
         input: "_content",
-        includes:"../_includes",
-        layouts:"../_layouts",
-        data:"../_data"
+        includes:"_includes",
+        layouts:"_layouts",
+        data:"_data"
```

Comment:
I moved the `_includes/`, `_layouts/`, and `_data/` folders into the `_content` directory.
Not saying there wasn't a way of making it work with a bunch of relative paths, but this seemed like an easier approach.